data/json/items/magazines.json

```diff
@@ -71,5 +70,235 @@
+    },{
+        "type" : "MAGAZINE",
+        "id" : "belt308",
+        "price" : 0,
+        "name" : "M13 7.62x51mm ammo belt",
+        "symbol" : "#",
+        "color" : "ltgray",
+        "description" : "An ammo belt consisting of metal linkages which disintegrate upon firing.  Designed for the M60 machine gun, and additionally compatible with the M240 machine gun.",
+        "material" : "steel",
+        "volume" : 0,
+        "weight" : 0,
+        "to_hit" : -4,
+        "rigid": false,
+        "ammo_type" : "308",
+        "capacity" : 200,
+        "count" : 100,
+        "reliability" : 7,
+        "flags": [ "NO_RELOAD", "NO_UNLOAD" ]
+    },{
+        "type" : "MAGAZINE",
+        "id" : "makarovmag",
+        "price" : 2300,
+        "name" : "Makarov PM magazine",
+        "symbol" : "#",
+        "color" : "ltgray",
+        "description" : "The standard magazine for the Pistolet Makarova, holds 8 rounds.",
+        "material" : "steel",
+        "volume" : 1,
+        "weight" : 60,
+        "to_hit" : -2,
+        "ammo_type" : "9x18",
+        "capacity" : 8,
+        "reliability" : 9,
+        "reload_time" : 100,
+        "flags": [ "COMPACT_MAG" ]
+    },{
+        "type" : "MAGAZINE",
+        "id" : "skorpion_61_mag",
+        "price" : 2300,
+        "name" : "Skorpion Vz. 61 magazine",
+        "symbol" : "#",
+        "color" : "ltgray",
+        "description" : "A standard 20-round magazine for the Skorpion Vz. 61, in .32 ACP.",
+        "material" : "steel",
+        "volume" : 2,
+        "weight" : 170,
+        "to_hit" : -2,
+        "ammo_type" : "32",
+        "capacity" : 20,
+        "reliability" : 8,
+        "reload_time" : 100,
+        "flags": [ "COMPACT_MAG" ]
+    },{
+        "type" : "MAGAZINE",
+        "id" : "skorpion_82_mag",
+        "price" : 2300,
+        "name" : "Skorpion Vz. 82 magazine",
+        "symbol" : "#",
+        "color" : "ltgray",
+        "description" : "A standard 20-round magazine for the Skorpion Vz. 82, in 9x18mm Makarov.",
+        "material" : "steel",
+        "volume" : 2,
+        "weight" : 170,
+        "to_hit" : -2,
+        "ammo_type" : "9x18",
+        "capacity" : 20,
+        "reliability" : 8,
+        "reload_time" : 100,
+        "flags": [ "COMPACT_MAG" ]
+    },{
+        "type" : "MAGAZINE",
+        "id" : "thompson_mag",
+        "price" : 3000,
+        "name" : "Thompson 20-round magazine",
+        "symbol" : "#",
+        "color" : "ltgray",
+        "description" : "A standard 20-round magazine for the Thompson submachine gun.",
+        "material" : "steel",
+        "volume" : 2,
+        "weight" : 170,
+        "to_hit" : -2,
+        "ammo_type" : "45",
+        "capacity" : 20,
+        "reliability" : 8,
+        "reload_time" : 100,
+        "flags": [ "COMPACT_MAG" ]
+    },{
+        "type" : "MAGAZINE",
+        "id" : "thompson_mag_30",
+        "price" : 4000,
+        "name" : "Thompson 20-round magazine",
```

Comment:
copy-paste typo
consider also *Thompson extended mag* for consistency with the other names